proteus/MeshAdaptPUMI/SizeField.cpp

```diff
@@ -17,13 +17,9 @@ static void SmoothField(apf::Field* f);
    thickness of refinement near the interface */
 static double isotropicFormula(double phi, double hmin, double hmax)
 {
-  static double const epsilon = 0.02;
-  phi = fabs(phi);
   double size;
-  if (fabs(phi) < epsilon)
+  if (fabs(phi) < 5.0*hmin)
```

Comment:
@zhang-alvin This makes a band of 10 elements of diamter hmin around the zero level set. See below for calling the smoothing so it doesn't reduce the size of the band. With this one could hope to take 5 steps at CFL=1 before the free surface leaves the refinement zone, but we have to be a little more careful than that. I'm using CFL=0.33 and an interface epsFact of 1.5 so the transition region is 3 elements wide.